index.bs

```diff
@@ -2021,7 +2031,31 @@ remove optional flag from the field.
 </div>
 
 <div algorithm>
-To <dfn>serialize as a list</dfn> given |iterable|, |max depth|, |child ownership|,
+To <dfn>serialize an Array-like</dfn> given |production|, |value|, |handle id|,
+|max depth|, |ownership type|, |known object|, |serialization internal map|, and |realm|:
```

Comment:
I would suggest the following change so that we keep the order of the general arguments similar to the other helper definitions and custom ones only affecting this helper at the front.

```suggestion
To <dfn>serialize an Array-like</dfn> given |production|, |handle id|, |value|,
|max depth|, |ownership type|, |known object|, |serialization internal map|, and |realm|:
```